File: dotnet/source/amp.eventing/streaming/StreamingEnvelopeHelper.cs
using cmf.bus;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using SEC = cmf.eventing.patterns.streaming.StreamingEnvelopeConstants;

namespace amp.eventing.streaming
{
    public class StreamingEnvelopeHelper
    {
        public static Envelope BuildStreamingEnvelope(string sequenceId, int position) 
        {
            Envelope envelope = new Envelope();
            envelope.Headers.Add(SEC.SEQUENCE_ID, sequenceId);
            envelope.Headers.Add(SEC.POSITION, position + "");
            return envelope;
        }
    }
}

File: java/amp-eventing/src/main/java/amp/eventing/streaming/StreamingEnvelopeHelper.java
package amp.eventing.streaming;

import cmf.bus.Envelope;

import static cmf.eventing.patterns.streaming.StreamingEnvelopeConstants.*;

public class StreamingEnvelopeHelper {

    public static Envelope buildStreamingEnvelope(String sequenceId, int position) {
        Envelope envelope = new Envelope();
        envelope.setHeader(SEQUENCE_ID, sequenceId);
        envelope.setHeader(POSITION, position + "");
        return envelope;
    }


}
